.travis.yml

```diff
@@ -1,30 +1,82 @@
-dist: trusty
 sudo: required
-language: python
+language: generic
+
 cache:
   directories:
     - $HOME/.ccache
 notifications:
   slack:
     on_success: never
     on_failure : never
-python:
-  - "2.7"
-  - "3.4"
-  - "3.5"
-  - "3.6"
-before_install:
-  - export PATH="/usr/lib/ccache:$PATH"
-  - ccache -M 256M && ccache -s
-  - sudo apt-get install libgmp-dev libmpfr-dev libmpfi-dev libmpc-dev cython texlive texlive-latex-extra latexmk
-  # note: sphinx in ubuntu is version 1.2.2 that does not support
-  # sphinx.ext.imgmath. We just install the last version.
-  - pip install sphinx
+
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+      language: python
+      python: 2.7
+
+    - os: linux
+      dist: trusty
+      language: python
+      python: 3.5
+
+    - os: linux
+      dist: trusty
+      language: python
+      python: 3.6
+
+    - os: osx
+      env: PYTHON_VERSION=2.7
+      osx_image: xcode6.4
+
+    - os: osx
+      env: PYTHON_VERSION=3.5
+      osx_image: xcode6.4
+
+    - os: osx
+      env: PYTHON_VERSION=3.6
+      osx_image: xcode6.4
+
 install:
-  - pip install --verbose .
+  - |
+    if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+      echo ""
```

Comment:
What is the point of `echo ""`?

Reply:
skipping a line in the trace. I think the point is readability.

Reply:
Readability of what? It does not help reading the script. And on travis you have access to the output of each command separatly.

Reply:
Readability of the log on travis console.
With the '|' char allowing multiple instructions, the whole block (if to fi in this instance) is considered as a single command by the logs.